src/mongo/dbtests/validate_tests.cpp

```diff
@@ -68,8 +68,10 @@ class ValidateBase {
         _client.createCollection(_ns);
         {
             AutoGetCollection autoGetCollection(&_opCtx, _nss, MODE_X);
-            _isInRecordIdOrder =
-                autoGetCollection.getCollection()->getRecordStore()->isInRecordIdOrder();
+            auto && col = autoGetCollection.getCollection();
+            if (col != nullptr) {
```

Comment:
Why is the collection "_nss" not always available? Can you check the DBClient::createCollection() result on line 68?

Reply:
@apocarteres we can address this validate_tests.cpp change in another pull request if you don't have the time to investigate why we need this conditional.

Reply:
sounds reasonable, i can exclude it from this PR for now